cloud-sql/mysql/sqlalchemy/connect_connector_auto_iam_authn.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START cloud_sql_mysql_sqlalchemy_auto_iam_authn]
+import os
+
+from google.cloud.sql.connector import Connector, IPTypes
+import pymysql
+
+import sqlalchemy
+
+
+# connect_with_connector_auto_iam_authn initializes a connection pool for
+# a Cloud SQL instance of MySQL using the Cloud SQL Python Connector
+# with Automatic IAM Database Authentication.
+def connect_with_connector_auto_iam_authn() -> sqlalchemy.engine.base.Engine:
```

Comment:
Method docstrings are preferred in this format:
```suggestion
def connect_with_connector_auto_iam_authn() -> sqlalchemy.engine.base.Engine:
    """ Initializes a connection pool for a Cloud SQL instance of MySQL.
    
    Uses the Cloud SQL Python Connector with Automatic IAM Database Authentication.
    """
```

Reply:
@dandhlee I will leave this as is for now and make a separate PR to go through and change all method comments into docstrings to have this change done in a consistent manner within this folder. That way all code snippets in the doc pages will be updated all at once.